index.js

```diff
@@ -85,7 +85,8 @@ exports.evm2wasm = function (evmCode, opts = {
   'stackTrace': false,
   'useAsyncAPI': false,
   'inlineOps': true,
-  'testName': 'temp'
+  'testName': 'temp',
+  'chargePerOp': false
```

Comment:
Please add the documentation line above.

Reply:
Done